modules/metastore/metastore.links.menu.yml

```diff
@@ -27,4 +27,4 @@ metastore.data_dictionary.settings_form:
   description: Data-Dictionary Settings
   parent: metastore.data_dictionary
   route_name: metastore.data_dictionary.settings
-  weight: 2
```

Comment:
If updating the menu, you'll need to add a post update hook.

Reply:
I think anyone updating code is familiar with clearing cache afterwards.